packages/vehicle-rental-overlay/src/index.js

```diff
@@ -0,0 +1,251 @@
+      </Popup>
+    );
+  };
+
+  renderStationAsCircle = (station, symbolDef) => {
+    let strokeColor = symbolDef.strokeColor || symbolDef.fillColor;
+    if (!station.isFloatingBike) {
+      strokeColor = symbolDef.dockStrokeColor || strokeColor;
+    }
+    return (
+      <CircleMarker
+        key={station.id}
+        center={[station.y, station.x]}
+        color={strokeColor}
+        fillColor={symbolDef.fillColor}
+        fillOpacity={1}
+        radius={symbolDef.pixels - (station.isFloatingBike ? 1 : 0)}
+        weight={1}
+      >
+        {this.renderPopupForStation(station)}
+      </CircleMarker>
+    );
+  };
+
+  renderStationAsHubAndFloatingBike = station => {
+    let icon;
+    if (station.isFloatingBike) {
+      icon = floatingBikeIcon;
+    } else {
+      const pctFull =
+        station.bikesAvailable /
+        (station.bikesAvailable + station.spacesAvailable);
+      const i = Math.round(pctFull * 9);
+      icon = hubIcons[i];
+    }
+    return (
+      <Marker icon={icon} key={station.id} position={[station.y, station.x]}>
+        {this.renderPopupForStation(station, !station.isFloatingBike)}
+      </Marker>
+    );
+  };
+
+  renderStationAsMarker = (station, symbolDef) => {
+    const color =
+      symbolDef && symbolDef.fillColor ? symbolDef.fillColor : "gray";
+    const markerIcon = getStationMarkerByColor(color);
+
+    return (
+      <Marker
+        icon={markerIcon}
+        key={station.id}
+        position={[station.y, station.x]}
+      >
+        {this.renderPopupForStation(station)}
+      </Marker>
+    );
+  };
+
+  renderStation = station => {
+    // render the station according to any map symbol configuration
+    const { mapSymbols } = this.props;
+
+    // no config set, just render a default marker
+    if (!mapSymbols) return this.renderStationAsMarker(station);
+
+    // get zoom to check which symbol to render
+    const zoom = this.props.leaflet.map.getZoom();
+
+    for (let i = 0; i < mapSymbols.length; i++) {
+      const symbolDef = mapSymbols[i];
+      if (symbolDef.minZoom <= zoom && symbolDef.maxZoom >= zoom) {
+        switch (symbolDef.type) {
+          case "circle":
+            return this.renderStationAsCircle(station, symbolDef);
+          case "hubAndFloatingBike":
+            return this.renderStationAsHubAndFloatingBike(station);
+          default:
+            return this.renderStationAsMarker(station, symbolDef);
+        }
+      }
+    }
+
+    // no matching symbol definition, render default marker
+    return this.renderStationAsMarker(station);
+  };
+
+  render() {
+    const { stations, companies } = this.props;
+    let filteredStations = stations;
+    if (companies) {
+      filteredStations = stations.filter(
+        station =>
+          station.networks.filter(value => companies.includes(value)).length > 0
+      );
+    }
+
+    if (!filteredStations || filteredStations.length === 0)
+      return <FeatureGroup />;
```

Comment:
Does the airbnb style guide really recommend not encapsulating multi-line conditionals in curly braces? Not sure I like that at all.

Reply:
On this style rule, it seems more like when you write a long 
`if (condition) thenStatement();` line, then es-lint recommends to put the thenStatement() into its own line. In that case would also add braces around thenStatement() to avoid confusion.